explorer/src/pages/Search/components/SearchWrapper/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from "react";
+
+interface SearchWrapperProps extends PropsWithChildren {
```

Comment:
I think we should use one concept for interfaces. You can place it in separate file.